### PR TITLE
Docs: Improve Landing Page

### PR DESCRIPTION
**Pull request by DrNickClarke**
### this is a draft but please review and leave feedback

* storage access docs moved to their own page
* code formatting now copy/paste friendly
* slight rewording, remove excessive use of !

### CODE REVIEW

File: docs/mkdocs/docs/index.md [hunk 1]
```diff
@@ -1,387 +1,495 @@
-![](images/FullWithBorder.png)
+ ![](images/FullWithBorder.png)
 
 ## What is ArcticDB?
 
-ArcticDB is an embedded/serverless database engine designed to integrate with Pandas and the Python Data Science ecosystem. ArcticDB enables 
-you to store, retrieve and process DataFrames at scale, backed by commodity object storage (S3-compatible storages and Azure Blob Storage).
+ArcticDB is a serverless DataFrame database engine designed for the Python Data Science ecosystem. 
+
+ArcticDB enables you to store, retrieve and process DataFrames at scale, backed by commodity object storage.
```
Comment (G-D-Petrov) [inline docs/mkdocs/docs/index.md:7]: I would keep the `(S3-compatible storages and Azure Blob Storage)` bit.
Not everyone knows what is an object store.

File: docs/mkdocs/docs/index.md [hunk 2]
```diff
@@ -1,387 +1,495 @@
...
-
-ArcticDB is designed for high throughput analytical workloads. It is _not_ a transactional database and as such is not a replacement for tools such as PostgreSQL.
+- **Fast**
+    - Process millions of rows per second
+    - Quick and easy to install: `pip install arcticdb`
+- **Flexible**
+    - No data schema is needed
+    - Supports streaming data ingestion
+    - Bitemporal - stores all previous versions of stored data
+    - As easy to get started as using files
```
Comment (G-D-Petrov) [inline docs/mkdocs/docs/index.md:20]: I would reword this line as something like:
`Easy to setup both locally and on cloud`

File: docs/mkdocs/docs/index.md [hunk 3]
```diff
@@ -1,387 +1,495 @@
...
+    - Scales from dev/research to production environments
+- **Familiar**
+    - ArcticDB is the world's simplest database
+    - Easy to learn for anyone with Python and Pandas experience
+    - Just you and your data - the cognitive overhead is very low.
+
+#### Transactions: Are They Overrated?
```
Comment (G-D-Petrov) [inline docs/mkdocs/docs/index.md:27]: The following sound better to me:
#### Transactions: Are They Overrated?
(This can be omitted)- Transactions can be expensive and slow
- Most analytical workflows can be constructed to run without relying on transactions
- So why pay the cost of transactions when they are often not needed?
- ArcticDB doesn't support transactions because it is designed for high throughput analytical workloads

File: docs/mkdocs/docs/index.md [hunk 4]
```diff
@@ -1,387 +1,495 @@
...
 
     ArcticDB also supports LMDB for local/file based storage - to use LMDB, pass an LMDB path as the URI: `adb.Arctic('lmdb://path/to/desired/database')`.
 
 To get started, we can import ArcticDB and instantiate it:
 
 ```python
->>> import arcticdb as adb
->>> ac = adb.Arctic(<URI>)
+import arcticdb as adb
+uri = "insert your uri string here"
```
Comment (G-D-Petrov) [inline docs/mkdocs/docs/index.md:60]: I would put a realistic looking uri here.
It really helps drive the point of how easy it is to setup Arctic

File: docs/mkdocs/docs/index.md [hunk 5]
```diff
@@ -1,387 +1,495 @@
...
-- **Familiar**: ArcticDB is the world's simplest database, designed to be immediately familiar to anyone with prior Python and Pandas experience.
-
-#### What is ArcticDB _not_?
-
-ArcticDB is designed for high throughput analytical workloads. It is _not_ a transactional database and as such is not a replacement for tools such as PostgreSQL.
+- **Fast**
+    - Process millions of rows per second
+    - Quick and easy to install: `pip install arcticdb`
+- **Flexible**
+    - No data schema is needed
```
Comment (vasil-pashov) [inline docs/mkdocs/docs/index.md:17]: This might be a bit misleading, as it's somewhat both right and wrong. As you note below append and update are not schemaless, but each write changes the schema. This requires us to start explaining dynamic vs static schema, which is advanced topic. I'm not sure we need it on the landing page. The landing page should be as small and concise as possible.

**Review by alexowens90 — CHANGES_REQUESTED** [review]

File: docs/mkdocs/docs/index.md (hunk 3 above)
Comment (vasil-pashov) [reply docs/mkdocs/docs/index.md:27]: I too have issue with this. 
* `Transactions: Are They Overrated?` the heading sounds a bit harsh. Almost like we're attacking a well established concept that's been in the software industry for years.
* We start with a question and don't give clear answer

File: docs/mkdocs/docs/index.md (hunk 5 above)
Comment (DrNickClarke) [reply docs/mkdocs/docs/index.md:17]: That is covered further down

File: docs/mkdocs/docs/index.md [hunk 6]
```diff
@@ -1,387 +1,495 @@
...
+    - Supports streaming data ingestion
+    - Bitemporal - stores all previous versions of stored data
+    - As easy to get started as using files
+    - Scales from dev/research to production environments
+- **Familiar**
+    - ArcticDB is the world's simplest database
```
[the remark this answers is not in the document]
Comment (DrNickClarke) [reply docs/mkdocs/docs/index.md:23]: True. That was already on the landing page actually. Has anyone seen a simpler database? I haven't

File: docs/mkdocs/docs/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (DrNickClarke) [reply docs/mkdocs/docs/index.md:27]: Good point. It was to improve on "What ArcticDB is not" which was weird imo.

File: docs/mkdocs/docs/index.md [hunk 7]
```diff
@@ -1,387 +1,495 @@
...
+- So why pay the cost of transactions when they are often not needed?
 
 ## Getting Started
 
-The below guide covers installation, setup and basic usage. More detailed information on advanced functionality such as _snapshots_ and _parallel writers_ can be found in the tutorials section.
+This section will cover installation, setup and basic usage. More details on basics and advanced features can be found in the [tutorials](tutorials/fundamentals.md) section.
 
 ### Installation
 
 ArcticDB supports Python 3.6 - 3.11. To install, simply run:
```
[the remark this answers is not in the document]
Comment (DrNickClarke) [reply docs/mkdocs/docs/index.md:40]: Might change is to 3.7 to 3.11 if we are thinking of retiring 3.6 support anyway

File: docs/mkdocs/docs/index.md [hunk 8]
```diff
@@ -1,387 +1,495 @@
...
 
-##### Connecting to a defined storage endpoint
+    The one exception where schemas are needed is in the case of functions that modify existing symbols: `update` and `append`. When modifying a symbol, the new data must have the same schema as the existing data. The schema here means the index type and the name, order, and type of each column in the DataFrame. In other words when you are appending new rows they must look like the existing rows. This is the default option and is called `static schema`.
 
-Connect to local storage (not AWS - HTTP endpoint of s3.local) with a pre-defined access and storage key:
+    However, if you need to add, remove or change the type of columns via `update` or `append`, then you can do that. You simply need to create the library with the `dynamic_schema` option set. See the `library_options` parameter of the ([`create_library`](api/arctic/#arcticdb.Arctic.create_library)) method.
+
+    So you have the best of both worlds - you can either enforce a static schema on your data so it cannot change by modifying operations, or allow it to be flexible.
```
Comment (vasil-pashov) [inline docs/mkdocs/docs/index.md:102]: I find this misleading as if you can change from static->dynamic (even though the next line explains that it's set at creation time)

File: docs/mkdocs/docs/index.md [hunk 9]
```diff
@@ -1,387 +1,495 @@
...
+df.head(5)
+```
+_output (the first 5 rows of the data)_
+```
+                     COL_0  COL_1  COL_2  COL_3  COL_4  COL_5  COL_6  COL_7  ...
+2000-01-01 05:00:00     18     48     10     16     38     34     25     44  ...
+2000-01-01 06:00:00     48     10     24     45     22     36     30     19  ...
+2000-01-01 07:00:00     25     16     36     29     25      9     48      2  ...
+2000-01-01 08:00:00     38     21      2      1     37      6     31     31  ...
+2000-01-01 09:00:00     45     17     39     47     47     11     33     31  ...
```
Comment (vasil-pashov) [inline docs/mkdocs/docs/index.md:131]: As a user I find it better when the examples are in a single snippet that I can copy/paste, rather than having to search the docs for the relative pieces and stich something working myself. I'd make one complete example creating a dataframe, creating arctic instance and library, storing the symbol and reading it back all in once place. And place the expected output below that.

File: docs/mkdocs/docs/index.md [hunk 10]
```diff
@@ -1,387 +1,495 @@
...
 
-You may want to restrict access for the ArcticDB library to a specific path within the bucket. To do this, you can use the `path_prefix` parameter:
+    When writing Pandas DataFrames, ArcticDB supports the following index types:
+    
+    * `pandas.Index` containing `int64` or `float64` (or the corresponding dedicated types `Int64Index`, `UInt64Index` and `Float64Index`)
+    * `RangeIndex` with the restrictions noted below
+    * `DatetimeIndex`
+    * `MultiIndex` composed of above supported types
+    
+    Currently, ArcticDB allows `append()`-ing to a `RangeIndex` only with a continuing `RangeIndex` (i.e. the appending `RangeIndex.start` == `RangeIndex.stop` of the existing data and they have the same `RangeIndex.step`). If a DataFrame with a non-continuing `RangeIndex` is passed to `append()`, ArcticDB does _not_ convert it `Int64Index` like Pandas and will produce an error.
```
Comment (vasil-pashov) [inline docs/mkdocs/docs/index.md:155]:
```suggestion
    Currently, ArcticDB allows `append()`-ing to a `RangeIndex` only with a continuing `RangeIndex` (i.e. the appending `RangeIndex.start` == `RangeIndex.stop` of the existing data and they have the same `RangeIndex.step`). If a DataFrame with a non-continuing `RangeIndex` is passed to `append()`, ArcticDB does _not_ convert it `pd.Index` like Pandas and will produce an error.
```

File: docs/mkdocs/docs/storage_access.md
[the remark this answers is not in the document]
Comment (DrNickClarke) [reply docs/mkdocs/docs/storage_access.md:1]: Ah yes I need to remove that. Thanks

[the remark this answers is not in the document]
Comment (DrNickClarke) [reply docs/mkdocs/docs/storage_access.md:1]: I did make it separate at one stage but decided to put it at the end instead

File: docs/mkdocs/docs/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jamesmunro) [reply docs/mkdocs/docs/index.md:23]: Well, embedded databases perhaps.  Simplest multi-user database?

File: docs/mkdocs/docs/index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jamesmunro) [reply docs/mkdocs/docs/index.md:33]: This page is part of the versioned docs, so make the same as the current ArcticDB version

File: docs/mkdocs/docs/index.md (hunk 3 above)
Comment (jamesmunro) [reply docs/mkdocs/docs/index.md]: My take: Atomicity, Consistency and Durability are not overrated.  Isolation is.  It would be helpful to explain a little.

File: docs/mkdocs/docs/index.md [hunk 11]
```diff
@@ -1,387 +1,495 @@
...
 
-- **Fast**: ArcticDB is incredibly fast, able to process millions of (on-disk) rows a second, and is very easy to install: `pip install arcticdb`!
-- **Flexible**: Supporting data with and without a schema, ArcticDB is also fully compatible with streaming data ingestion. The platform is bitemporal, allowing you to see all previous versions of stored data.
-- **Familiar**: ArcticDB is the world's simplest database, designed to be immediately familiar to anyone with prior Python and Pandas experience.
-
-#### What is ArcticDB _not_?
-
-ArcticDB is designed for high throughput analytical workloads. It is _not_ a transactional database and as such is not a replacement for tools such as PostgreSQL.
+- **Fast**
+    - Process millions of rows per second
```
[the remark this answers is not in the document]
Comment (jamesmunro) [reply docs/mkdocs/docs/index.md:14]: Yes, all the time, across consumers and up to 100M for a single consumer.

**Review by jamesmunro — APPROVED** [review]
